README.md

```diff
@@ -456,7 +456,7 @@ GCE_VM_NAME=minigo-etpu-test
 GCE_ZONE=us-central1-f
 
 # Clone the Minigo Github repository:
-git clone https://github.com/tensorflow/minigo
+git clone --depth 1 https://github.com/tensorflow/minigo
```

Comment:
can we keep this one as is? It's nice to have full history when developing.

Reply:
@sethtroisi  , this section tutorial is for retraining a model on GCP , Only being stored in Cloud bucket to be used for model training .

That why i think no one is going to look at history .

Reply:
ahh. I had just assumed :) Thanks for the explination.